formSkeleton/forms/Vaststelling-gemeentelijk-beleidskader-gemeentewegen/form.ttl

```diff
@@ -0,0 +1,161 @@
+
+
+########### Vaststelling gemeentelijk beleidskader (gemeentewegen) ###########
+
+# Conditionally add the form fields to the "type dossier" field
+fields:0827fafe-ad19-49e1-8b2e-105d2c08a54a form:hasConditionalFieldGroup fields:14cf00b9-af35-4dc7-9a6e-78c7f9b98d2a.
+
+# Only add the fields if the user selected the "Vaststelling gemeentelijk beleidskader (gemeentewegen)" option
+fields:14cf00b9-af35-4dc7-9a6e-78c7f9b98d2a a form:ConditionalFieldGroup ;
+    mu:uuid "14cf00b9-af35-4dc7-9a6e-78c7f9b98d2a";
+    form:conditions
+      [ a form:SingleCodelistValue ;
+        form:grouping form:Bag ;
+        sh:path rdf:type ;
+        form:conceptScheme <http://lblod.data.gift/concept-schemes/71e6455e-1204-46a6-abf4-87319f58eaa5> ;
+        form:customValue <https://data.vlaanderen.be/id/concept/BesluitType/f1fd8f88-95b0-4085-b766-008b5867d992>
+      ] ;
+    form:hasFieldGroup fieldGroups:94b97172-f358-4ffd-b8e7-81898551a97c .
+
+# Fields that should always be added to the current form
+fieldGroups:94b97172-f358-4ffd-b8e7-81898551a97c a form:FieldGroup ;
+    mu:uuid "94b97172-f358-4ffd-b8e7-81898551a97c" ;
+
+    form:hasField
+      # TODO Can we reuse this, or should we create a new field and concepts?
+      ###Gaat het over een voorlopige of definitieve vaststelling?###
+      fields:b9d831c5-da21-40d6-aac8-65feb4783d76,
```

Comment:
I'm not sure if I can reuse the field (and concept values) like this. The label is generic enough so I assumed it was ok, but that might be wrong.

Reply:
The concept-scheme encapsulates "type vaststelling voor besluiten" BUT the behavior is unique. If you re-use it, I'm concerned we'll start to see the newly added `form:hasConditionalFieldGroup`s appear across other sub-forms.